.github/workflows/weekly-doc-check.yml

```diff
@@ -10,7 +10,55 @@ on:
   workflow_dispatch:
 
 jobs:
+  rust-build:
+    runs-on: ubuntu-22.04
+    defaults:
+      run:
+        working-directory: rust
+    env:
+      CARGO_TERM_COLOR: always
+    steps:
+
+      - name: Free disk space
+        uses: jlumbroso/free-disk-space@v1.3.1
+        with:
+          tool-cache: false     # no: will remove rust
+          android: true         # 8.7GiB
+          dotnet: true          # 1.6GiB
+          docker-images: true   # 3.2GiB
+          haskell: false        # no: little/no benefit
+          large-packages: false # no: slow
+          swap-storage: false   # no: having swap is useful
+
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Cache Rust dependencies
+        uses: actions/cache@v4
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            rust/target
+          key: ${{ runner.os }}-cargo-${{ hashFiles('rust/Cargo.toml') }}
+          restore-keys: ${{ runner.os }}-cargo-
+
+      - name: Check Rust version
+        run: cargo --version
+
+      - name: Build
+        # We only need "bindings", but build fails without "untrusted".
+        # TODO: All combos of features should work: https://github.com/opendp/opendp/issues/1343
+        run: cargo build --verbose --features bindings,untrusted
```

Comment:
To only generate bindings, then run `cargo check` instead.

Reply:
Maybe I'm missing something, but that [doesn't seem to work by itself](https://github.com/opendp/opendp/actions/runs/10180763679/job/28159554880#step:7:189). Are there additional arguments?

Reply:
I was thinking to just substitute `build` with `check`! It still needs features set.

Reply:
(Restored `cargo build` in the latest commit, but if there's something better, great!)